### ISSUE TITLE
Add Orientation Estimation Task

### ISSUE THREAD

**Issue by rjwalden** [Rosenblatt-LLC/kolmogorov#8]

**Comment by rjwalden** [comment]
Cannot be reviewed until orientation data is added to the dataset.

**Comment by rjwalden** [comment]
Modifications:

`### def build_finetune_model ###`
`classification = tf.keras.layers.Dense(num_classes, activation='softmax', name='classification')(x)`
`orientation = tf.keras.layers.Dense(3, activation='sigmoid', name='orientation')(x) `
`return tf.keras.Model(inputs=base_model.input, outputs=[classification, orientation])`

`### def train ###`
`model.compile(optimizer=opt, 
                  metrics=['accuracy'], 
                  loss={'classification': 'categorical_crossentropy', 'orientation': 'mse'},
                  loss_weights={'main_output': 0.5, 'aux_output': 0.5})`
